tests/Doctrine/Tests/ORM/Functional/Ticket/GH6884Test.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Event\LifecycleEventArgs;
+use Doctrine\ORM\Events;
+use Doctrine\Tests\OrmFunctionalTestCase;
+use stdClass;
+
+/**
+ * @group 6884
+ */
+final class GH6884Test extends OrmFunctionalTestCase
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->_schemaTool->createSchema(
+            [
+                $this->_em->getClassMetadata(GH6884Person::class)
+            ]
+        );
+
+        $builder = $this->getMockBuilder(stdClass::class);
```

Comment:
Please use `createPartialMock() `, it's simpler

Reply:
This method was not working for me on my dev environment, I might have not had my composer updated, I'll take a peak later today